common/src/timestamp.rs

```diff
@@ -0,0 +1,37 @@
+use core::convert::TryFrom;
+use displaydoc::Display;
+use serde::{Deserialize, Serialize};
+
+#[derive(PartialEq, Eq, Debug, Display, Clone, Serialize, Deserialize)]
+#[repr(u32)]
+pub enum TimestampResultCode {
```

Comment:
Any reason this isn't inside watcher_db? or mc-api/fog api? Maybe some no std issue?
It feels to me like mc-common is not the place for this.

Reply:
That's my main question for this PR - where does this belong? It can't live in watcher, indeed, because of a no_std issue. But maybe there's another way to get around that? I read the README for common, which says it's meant for code shared inside and outside of enclaves, so that's why I put it here.

Reply:
Probably a watcher-api crate :/

Reply:
Ok - that's a good solution. I'll do that!

Reply:
Done! Made watcher-api with no_std, and moved TimestampResultCode there - testing now with the changes in internal to make sure it builds for all the enclaves.